docs/sources/writing-guide/topic-types/tutorial/index.md

```diff
@@ -96,6 +96,58 @@ For example, the [Play with Grafana Mimir](/tutorials/play-with-grafana-mimir/)
 
 If getting access to the tutorial data is complex, include the instructions in the steps of the tutorial. If getting access to the data is straightforward, include it in the "Before you begin" section.
 
+## Publish your tutorial
+
+Your tutorial source is stored in your project repo in a `tutorials` folder and mounted to the tutorials repo so that it can be displayed on the [Tutorials](/tutorials) page. The source is stored in your project repo to make it easy for team members to review and edit the content. 
+
+Follow the steps below to hide the tutorial from your project's table of contents and to display it on the Tutorials page. 
+
+### Hide your tutorial from your Table of Contents
+
+Before completing these steps, you must create a `tutorials` directory under `sources` and add your tutorial into its own subdirectory as described in the [Write a tutorial topic](#write-a-tutorial-topic) section.
+
+- Hide your tutorial from your documentation table of contents by integrating the following to the `$.cascade` field in the YAML metadata of your `sources/_index.md` file. 
+
+    ```yaml
+    cascade:
+    - _target:
+        path: /docs/grafana-cloud/tutorials/**
+      _build:
+        list: false
+        render: false
+    - _target:
+        path: /docs/grafana-cloud/**
+    ```
+
+    > Create the `cascade` field if it does not exist. Substitute your repo for `grafana-cloud` in this example.
+
+### Add your tutorial to the Tutorials page
+
+> Note: This procedure is for writers who have permissions to update the Grafana website repo.
+
+To add your tutorial to the Tutorials page:
+
+1. Integrate the following to the `$.manual_mounts` field in the `config/_default/config.yaml` file in the website repo:
+
+    ```yaml
+    manual_mounts:
+      - source: content/docs/grafana-cloud/tutorials/k8s-monitoring-app
+        target: content/tutorials/k8s-monitoring-app
+    ```
+
+    > Create the `manual_mounts` field if it does not exist. Substitute the source and target with your tutorial's path and name.
+
+1. Specify the level and type of your tutorial (substitute the target path of your tutorial):
```

Comment:
Maybe divide this into 2 steps? Like:

1. Specify the tutorial's `level` and `type`.
1. Add the target path to the tutorial content in the `page` attribute.